Model/Notification.php

```diff
@@ -156,15 +158,15 @@ public function getLastVersion()
             $url = self::CHECK_VERSION_URL;
             $client = $this->curlFactory->create();
 
-            $client->write(\Zend_Http_Client::GET, $url, '1.1');
+            $client->write(Request::METHOD_GET, $url, '1.1');
             $responseBody = $client->read();
             $client->close();
 
-            $responseCode = \Zend_Http_Response::extractCode($responseBody);
+            $responseCode = $this->extractCodeFromResponse($responseBody);
             if ($responseCode !== 200) {
                 return false;
             }
-            $body = \Zend_Http_Response::extractBody($responseBody);
+            $body = Response::getBody($responseBody);
```

Comment:
getBody is not static method, it causes error.

Reply:
with this PR we can't open fastly config in administration (system section)

Reply:
@dpotkoc fixed the issue.